src/main/java/com/puppycrawl/tools/checkstyle/api/FileText.java

```diff
@@ -171,7 +167,7 @@ public FileText(FileText fileText) {
      * @param lines the lines of the text, without terminators
      * @throws NullPointerException if the lines array is null
      */
-    private FileText(File file, List<String> lines) {
+    public FileText(File file, List<String> lines) {
         final StringBuilder buf = new StringBuilder();
         for (final String line : lines) {
             buf.append(line).append('\n');
```

Comment:
@rnveach 
Created the issue https://github.com/checkstyle/checkstyle/issues/4545 .

@romani 
Please, take a look and approve.